blocks/v2-hero/v2-hero.css

```diff
@@ -0,0 +1,82 @@
+/* stylelint-disable selector-class-pattern */
+.v2-hero {
+  position: relative;
+
+  /* height: 798px; */
+  height: 588px;
+  overflow: hidden;
+}
+
+.v2-hero__image {
+  position: absolute;
+  object-fit: cover;
+  height: 100%;
+  object-position: center;
+  width: 100%;
+}
+
+.v2-hero__content-wrapper {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-end;
+}
+
+.v2-hero__content {
+  padding: 40px 16px;
+  display: flex;
+  flex-direction: column;
+  max-width: 1040px;
+}
+
+.v2-hero__heading {
+  margin: 24px 0;
+  color: var(--c-primary-white);
+  font-family: var(--ff-headline-medium);
+  font-size: 28px;
+  font-style: normal;
+  line-height: 120%;
```

Comment:
`1.15;` instead of `120%`

Reply:
On the mobile design, it is `1.2`, but `1.15` for the desktop. I missed the desktop one.

Reply:
I updated as it is on the design. 1.2 on mobile and 1.15 on desktop. If you want me to change it please let me know